### PR TITLE
getting started fixes, prereqs, links to table example

### PR DESCRIPTION
**Pull request by alexsb**

### CODE REVIEW

File: contributors/prereqs.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ organization on Github. If you haven't used git itself before, please read throu
 [this tutorial](https://git-scm.com/docs/gittutorial),
 or try this [interactive one](https://try.github.io/).
 
+You will also need to set-up SSH access to github. Here's [how to do that](https://help.github.com/articles/connecting-to-github-with-ssh/).
```
Comment (sgratzl) [inline contributors/prereqs.md]: you don't need to

Comment (sgratzl) [reply contributors/prereqs.md]: the generator always asks you if you want to use SSH clone or not

File: contributors/working_with_datastructures.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: documentation
+title:  Working with Datastructures
+order: 2
+---
+
+## 
+
+Check out the example on how to work with a table [here](https://github.com/Caleydo/lineage/blob/master/src/dataExplorations.ts).
```
Comment (sgratzl) [inline contributors/working_with_datastructures.md]: I don't like this approach linking to a repository which code changes. I would prefer a clean hello world repository without anything else

File: contributors/getting_started.md [hunk 3]
```diff
@@ -154,12 +154,12 @@ Create the client side as follows:
 6. Run `yo phovea:init-app` and follow the prompts. This initializes a new client app.
 7. `cd ..`
 
-If you have an existing phovea client side app, you can just clone that repository into this directory. 
+If you have an existing phovea client side app, you can just clone that repository into this directory. If your existing repository was a client-only repository, make sure to update the client only flag in `"clientOnly": false,`
```
Comment (sgratzl) [inline contributors/getting_started.md]: this is a magic hint, that no ones understands just from this sentence

File: contributors/working_with_datastructures.md (hunk 2 above)
Comment (alexsb) [reply contributors/working_with_datastructures.md]: The plan is to create a new demo repository, but it's here now, and this is better than not having it.

Comment (mstreit) [reply contributors/working_with_datastructures.md]: @alexsb, we can point to https://github.com/Caleydo/phovea_demos/blob/master/src/UsingTable.ts now, right?

**Comment by mstreit** [comment]
Looks good as far as I can tell. Will merge it now. Thanks @alexsb for taking the time to contribute to the documentation!

**Pull request merged**